setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 100
+ignore =
+    # E731: do not assign a lambda expression
+    E731
```

Comment:
I'd just fix those since there are only 2 instances left - you already did the most annoying work by fixing the weird indentation :)